style/values/generics/image.rs

```diff
@@ -28,7 +28,7 @@ pub enum GenericImage<G, ImageUrl, Color, Percentage, Resolution> {
     /// `none` variant.
     None,
     /// A `<url()>` image.
-    Url(ImageUrl),
+    Url(Box<ImageUrl>),
```

Comment:
I don't think this should be boxed. `ImageUrl` (which is a type alias for `CssUrl`) already contains an `Arc`, so boxing here is double boxing.

Reply:
@nicoburns have removed boxing from here but the computed image size still remains at 24 bytes. The specified image size got reduced to 16 though. Any pointers as to why that would be happening?

Reply:
This seems to be because `ComputedUrl` (https://github.com/servo/stylo/blob/main/style/servo/url.rs#L209) is still 16 bytes (8 + enum tag) which becomes 24 when combined with the tag for `Image`.

Reply:
Oh, got it. So, should we aim to get it to 16 in this PR or should we take it up in another one?